core/env.js

```diff
@@ -85,14 +85,21 @@
   
   /** Get a list of boards that we know about */
   function getBoardList(callback) {
-    Espruino.Core.Utils.getJSONURL(JSON_DIR + "boards.json", function(boards){
+    var jsonDir = Espruino.Config.BOARD_JSON_URL || DEFAULT_JSON_DIR;
```

Comment:
That's fine, but `Espruino.Config.BOARD_JSON_URL || DEFAULT_JSON_DIR` can just be `Espruino.Config.BOARD_JSON_URL`, since `Espruino.Config.BOARD_JSON_URL` won't be empty unless the user has actually deleted the setting (in which case a lot of things will break!).